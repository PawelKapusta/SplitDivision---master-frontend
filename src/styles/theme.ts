import { DefaultTheme } from "styled-components";

export const theme: DefaultTheme = {
    palette: {
        white: "#fff",
        black: "#000",
        background1: "#0F1624",
        gold: "#eaaf57",
    },
    breakpoints: {
        sm: "screen and (max-width: 640px)",
        md: "screen and (max-width: 768px)",
        lg: "screen and (max-width: 1024px)",
        xl: "screen and (max-width: 1280px)",
    },
};
