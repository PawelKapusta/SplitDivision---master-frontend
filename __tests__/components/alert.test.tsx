import { render, screen } from "@testing-library/react";
import { ThemeProvider } from "styled-components";
import Alert from "@components/alert/index";
import { mockTheme } from "../mocks/theme";

describe("alert component", () => {
    it("renders correctly", () => {
        const { container } = render(
            <ThemeProvider theme={mockTheme}>
                <Alert message="Test alert message" type="success" />
            </ThemeProvider>,
        );
        expect(screen.getByText("Test alert message")).toBeInTheDocument();
        expect(container).toMatchSnapshot();
    });
});
