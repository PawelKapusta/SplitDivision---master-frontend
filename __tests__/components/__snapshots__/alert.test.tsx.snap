// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`alert component renders correctly 1`] = `
<div>
  <div
    class="sc-49f0a3d4-0 dEbtqz"
    type="success"
  >
    <span
      class="sc-49f0a3d4-1 bFwfBF"
      type="success"
    >
      <img
        alt="Alert Icon"
        src="/icons/success-icon.svg"
      />
    </span>
    Test alert message
  </div>
</div>
`;
